build_web_compilers/lib/src/common.dart

```diff
@@ -18,8 +18,7 @@ String defaultAnalysisOptionsArg(ScratchSpace scratchSpace) =>
 
 // TODO: better solution for a .packages file, today we just create a new one
 // for every kernel build action.
-Future<File> createPackagesFile(
-    Iterable<AssetId> allAssets, ScratchSpace scratchSpace) async {
```

Comment:
I guess `scratchSpace` was never used? Is there a lint we can turn on for that?

Reply:
It's a tough lint to write - there would be false positives when you write a function that will be used as a tear-off and must fit a certain signature.

Reply:
Yeah I thought about that use case but if we had a pattern of naming unused parameters with underscores that could resolve the issue?